coalahtml/_coalahtml/index.html

```diff
@@ -9,9 +9,9 @@
     <link rel='stylesheet' type='text/css'
```

Comment:
Not sure about changing the entry point in gh-pages ...
But I think gh-pages becomes useless once we get a coala website - which I am ok with.

For gh-pages we probably want a different system completely if we're not using bower. We could just run coala on some test project and the folder it creates with `--dir` can be duped there ?